addons/payment_alipay/i18n/payment_alipay.pot

```diff
@@ -110,3 +110,10 @@ msgstr ""
 #, python-format
 msgid "received invalid transaction status: %s"
 msgstr ""
+
+#. module: payment_alipay
+#: model_terms:ir.ui.view,arch_db:payment_alipay.payment_provider_form
+msgid ""
+"This provider is deprecated.\n"
+"                    Consider disabling it and moving to <strong>Asiapay</strong>."
+msgstr ""
```

Comment:
This term (as well as the same term for all the other payment providers) were added in the wrong spot in the file